projects/packages/waf/src/brute-force-protection/class-blocked-login-page.php

```diff
@@ -51,8 +54,8 @@ class Jetpack_Protect_Blocked_Login_Page {
 	/**
 	 * URL to support page
 	 *
-	 * @deprecated 8.5.0 Use Jetpack_Protect_Blocked_Login_Page::get_help_url()
-	 * @see Jetpack_Protect_Blocked_Login_Page::get_help_url()
+	 * @deprecated 8.5.0 Use Brute_Force_Protection_Blocked_Login_Page::get_help_url()
```

Comment:
We'll need to update the `8.5.0` here, I'm assuming that is referencing the Jetpack version number. That said, I wonder if the deprecated tag is relevant at all now that the code is in a different class outside of the Jetpack plugin.

This constant is unused by the monorepo, so I'm curious why we're tagging it. Something like this would typically be relevant when the value is being used by other Jetpack packages for example.

We might consider removing the `HELP_URL` constant entirely.